src/js/relayclient/utils.js

```diff
@@ -92,7 +92,7 @@ module.exports = {
         else
             signed = hash
         let keyHex = "0x" + Buffer.from(privKey).toString('hex')
-        const sig_ = EthCrypto.sign(keyHex, signed)
+        const sig_ = sign(keyHex, signed)
```

Comment:
I believe that we can actually do `const sig_ = ethUtils.ecsign(signed, keyHex)` and remove eth-crypto altogether.